apps/pie-docs/src/tokensMetadata.json

```diff
@@ -1501,6 +1501,52 @@
     }
   },
   "elevation": {
+    "global": {
+      "box-shadow-00": {
+        "displayName": "Box shadow 00",
+        "description": "0,0,0,0 rgba(0,0,0,0)<br /> 0,0,0,0 rgba(0,0,0,0)<br /> 0,0,0,0 rgba(0,0,0,0)"
+      },
+      "box-shadow-01": {
+        "displayName": "Box shadow 01",
```

Comment:
the display names are Elevation 0x in designs, are you sure we should use box shadow 0x instead?